app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(copy_command: str) -> None:
+    copy_commands = copy_command.split()
+    old_name = copy_commands[1]
+    new_name = copy_commands[2]
```

Comment:
You can simplify this
```suggestion
    command, old_name, new_name = copy_command.split()
```